public/apps/configuration/panels/test/get-started.test.tsx

```diff
@@ -81,6 +83,7 @@ describe('Get started (landing page)', () => {
           navigation={{} as any}
           params={{} as any}
           config={config as any}
+          securityPluginStartDeps={{}}
```

Comment:
This variable replaces `navigation`. Can you change the test to reflect the change in name from `navigation` -> `depsStart` or `securityPluginStartDeps`

Reply:
We don't really use any of the deps in other tests, other than the new ones that I added to verify the datasourceEnabled functionality - do you have any sugestions on testing the change/rename, other than the existing tests passing?

Reply:
The existing tests passing will be fine since its a refactor of a variable and no functionality is changed.

I left a comment here because `navigation` and `securityPluginStartDeps` are both being passed in as props, but this PR shows that `navigation` is being renamed to `securityPluginStartDeps`, so this should be modified to remove references to the old name `navigation`.

Reply:
Oh ok, I misunderstood what you meant, sure removing the old navigation variable makes sense to me